javascript/pop_up.js

```diff
@@ -0,0 +1,161 @@
+// Instantiate project object
+
+console.log('HELLO');
```

Comment:
- [ ] HELLO 😄. Please it's recommended to use JS best practices. So I recommend removing all the console.log in your file